app/assets/stylesheets/application.css.scss

```diff
@@ -28,3 +28,21 @@
 .downvote {
   color: red;
 }
+
+#foto,
+#foto img {
+  background: #1abc9c;
```

Comment:
Color literals like `#1abc9c` should only be used in variable declarations; they should be referred to via variable everywhere else.